src/MessageWriter.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\AzureEventHubWriter;
+
+use Keboola\AzureEventHubWriter\Exception\ApplicationException;
+use React\Stream\WritableStreamInterface;
+use Throwable;
+
+class MessageWriter
```

Comment:
PHP posiela spravy do NodeJs procesu cez pipe - osobitny FD, ... tato trieda to riesi.